views/helpers.go

```diff
@@ -0,0 +1,86 @@
+package views
+
+import (
+	"github.com/gorilla/sessions"
+	"github.com/ystv/web-auth/user"
+)
+
+type (
+	// Context is a struct that is applied to the templates.
+	Context struct {
+		// Message is used for sending a message back to the user trying to log in, might decide to move later as it may not be needed
+		Message string
+		// MsgType is the bulma.io class used to indicate what should be displayed
+		MsgType string
```

Comment:
Is this different to the `Notification` type in `views/forgot.go`?

Reply:
Maybe, this was a legacy component that I think Rhys wrote, it worked so it just stayed

Reply:
I may change it later but is working